### DISCUSSION TITLE
Using teleport

### DISCUSSION THREAD

**Discussion by pdmshrestha** [histoire-dev/histoire#541] [Q&A] [answered]
Hi, is there a workaround for adding components to story which uses teleport?

**Comment by pdmshrestha** [comment, marked as answer]
By adding required element to dom using global setup helped me. https://histoire.dev/guide/vue3/app-setup.html#global-setup

**Reply by adriangube** [reply]
@pdmshrestha How did you solve the issue? I'm facing a similar one. I'm trying to use a teleport to the `body` but it's not working. Any ideas?

**Reply by pdmshrestha** [reply]
It worked me by adding the element using addWrapper function.
```ts

export const setupVue3 = defineSetupVue3(({ app, addWrapper }) => {
  addWrapper(HistoireApp);
});

// HistoireApp.vue
<template>
  <div class="histoireapp">
    <div id="my-teleport-target"></div>

    <slot></slot>
  </div>
</template>


```
